packages/claude-hooks/src/voices/list-voices.ts

```
@@ -0,0 +1,352 @@
+  claude-hooks-list-voices -f json                   # Output as JSON
+  claude-hooks-list-voices --preview                 # Enable voice previews
+  claude-hooks-list-voices --api-key sk-...         # Use specific API key
+
+ENVIRONMENT VARIABLES:
+  OPENAI_API_KEY           OpenAI API key
+  ELEVENLABS_API_KEY       ElevenLabs API key
+        `)
+        process.exit(0)
+
+      default:
+        if (arg.startsWith('--preview:')) {
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)